webhook/resources/node/validator.go

```diff
@@ -142,3 +142,17 @@ func (n *nodeValidator) Update(request *admission.Request, oldObj runtime.Object
 
 	return nil
 }
+
+func isNodeDiskSpecAndStatusSynced(node *longhorn.Node) bool {
+	if len(node.Spec.Disks) != len(node.Status.DiskStatus) {
+		return false
+	}
+
+	for diskName := range node.Spec.Disks {
+		if _, ok := node.Status.DiskStatus[diskName]; !ok {
```

Comment:
Need to check the path as well

Reply:
`Path`? DiskStatus does not have `Path`.

Reply:
Oh, sorry. You are right